File: Geomviz.jl/src/vga.jl
encode(a::Multivector{3,1}) = rig(
	"Arrow Vector",
	location=(0,0,0),
	"Vector"=>Vector(a.comps),
)

encode(a::Multivector{3,2}) = rig(
	"Circle 2-blade",
	location=(0,0,0),
	"Normal"=>Vector(rdual(a).comps),
	"Radius"=>sqrt(sqrt(abs(a⊙a))),
)

File: python/geomviz/vga.py
from math import sqrt

def vector(mv):
	return {
		'rig_name': "Arrow Vector",
		'rig_parameters': {
			"Vector": mv.value[1:4].tolist()
		}
	}

def bivector(mv):
	return {
		'rig_name': "Circle 2-blade",
		'rig_parameters': {
			"Normal": mv.value[5:8].tolist(),
			"Radius": sqrt(abs(mv))
		}
	}